XML/Other/ReplaceableTestParameters.xml

```diff
@@ -151,7 +151,11 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>INFINIBAND_INTEL_MPI_SETTINGS_IB</ReplaceThis>
-		<ReplaceWith>-env I_MPI_FABRICS ofi -env SECS_PER_SAMPLE=600</ReplaceWith>
+		<ReplaceWith>-env I_MPI_FABRICS shm:dapl -env I_MPI_DAPL_PROVIDER=ofa-v2-ib0 -env SECS_PER_SAMPLE=600</ReplaceWith>
+	</Parameter>
+	<Parameter>
+		<ReplaceThis>INFINIBAND_INTEL_MPI_SETTINGS_ND</ReplaceThis>
+		<ReplaceWith>-env I_MPI_FABRICS dapl -env I_MPI_DAPL_PROVIDER=ofa-v2-ib0 -env SECS_PER_SAMPLE=600</ReplaceWith>
```

Comment:
Is there a place that we can review the CI test results?

Reply:
I am not sure I am fully understanding your question. LISAv2 is the test automation not the CI infra. LISAv2 keeps the logs in stdout and zip file. CI can display it by design. We can discuss offline.